meta-lmp-base/recipes-support/docker-image-preload/docker-image-preload/docker-image-preload.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Docker container images preloading
+
+After=local-fs.target
+Before=docker.service
+
+ConditionPathExists=/var/sota/import/installed_versions
```

Comment:
IIRC this file is removed by aktualizr once it starts for the first time, so probably also good to add a dependency on aktualizr-lite (Before).

Reply:
It doesn't bring any value.